### DISCUSSION TITLE
how to remove a jump shortcut

### DISCUSSION THREAD

**Discussion by AbhimanyuAryan** [ajeetdsouza/zoxide#566] [Q&A] [answered]
whenever I write `z stipple` it jumps to `~/.julia/dev/StipplePlotly` instead I want it to jump to `~/.julia/dev/Stipple`. How can I do that?

**Comment by xfzv** [comment, marked as answer]
```console
zoxide remove ~/.julia/dev/StipplePlotly
zoxide add ~/.julia/dev/Stipple
```

Referencess:
- [zoxide-remove.1](https://github.com/ajeetdsouza/zoxide/blob/main/man/man1/zoxide-remove.1)
- [zoxide-add.1](https://github.com/ajeetdsouza/zoxide/blob/main/man/man1/zoxide-add.1)
